bgpd/bgp_io.c

```diff
@@ -191,6 +191,8 @@ static int bgp_process_reads(struct thread *thread)
 	if (CHECK_FLAG(status, BGP_IO_TRANS_ERR)) {
 		/* no problem; just don't process packets */
 		more = false;
+		;
+		;
```

Comment:
are you trying to make some script somewhere happy?

Reply:
Yes, this is a testing PR